### PR TITLE
Package updates

### PR DESCRIPTION
**Pull request by timbussmann**
Connects to Particular/PlatformDevelopment#757

@andreasohlund @mikeminutillo please do a sanity check
(did not bump the nuspec dependency for core as discussed with simon and andreas)

fixed a test failing on my machine. Does not yet have a TC build

### CODE REVIEW

File: src/ServiceControl.Plugin.Nsb6.Heartbeat.AcceptanceTests/When_trying_to_parse_message_using_core_json_serializer.cs [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
     using NServiceBus.AcceptanceTesting;
     using NUnit.Framework;
     using System.Threading.Tasks;
     using NServiceBus.AcceptanceTests.EndpointTemplates;
-    using ServiceControl.Plugin.Heartbeat.Messages;
+    using Plugin.Heartbeat.Messages;
 
     public class When_trying_to_parse_message_using_core_json_serializer
     {
+        private const string EndpointName = "Heartbeat.Nsb6.HeartbeatEndpoint";
```
Comment (andreasohlund) [inline src/ServiceControl.Plugin.Nsb6.Heartbeat.AcceptanceTests/When_trying_to_parse_message_using_core_json_serializer.cs:13]: pbot sync + code cleanup? (or do you plan to handle that separately)

Comment (timbussmann) [inline src/ServiceControl.Plugin.Nsb6.Heartbeat.AcceptanceTests/When_trying_to_parse_message_using_core_json_serializer.cs:13]: will do a separate PR for that

**Comment by andreasohlund** [comment]
LGTM
